inappmessaging/src/main/java/com/rakuten/tech/mobile/inappmessaging/runtime/data/responses/ping/CampaignData.kt

```diff
@@ -28,8 +28,14 @@ internal data class CampaignData(
     private val infiniteImpressions: Boolean = false
 ) : Message {
 
-    @SerializedName("timesClosed")
-    internal var timesClosed = 0
+    @SerializedName("impressionsLeft")
+    override var impressionsLeft: Int? = null
```

Comment:
Why is this initialized to null? Same with `isOptedOut`?

Reply:
Yes, same case